app/src/main/res/values-night/themes.xml

```diff
@@ -0,0 +1,34 @@
+<resources>
+
+    <!-- App Theme -->
+
+    <style name="AppDayNightTheme" parent="AppBaseTheme">
+		<item name="colorControlNormal">@color/drawableTintDark</item>
```

Comment:
intentation wrong

Reply:
Good catch, changed in `values-v29/themes.xml` too.